Makefile

```diff
@@ -3,50 +3,65 @@ VERSION = 0.0.1
 CC      = clang
 SA      = scan-build
 
-DIR 	:= $(pwd)
+DIR 	:= $(shell pwd)
```

Comment:
shell is a keyword in make that runs the argument in your shell. That line is basically command substitution in shell languages.